### ISSUE TITLE
Can't see character name in player customisation menu

### ISSUE THREAD

**Issue by fomalsd** [unitystation/unitystation#4047]
<img width="254" alt="Scr 2020-04-26 в 22 12 34" src="https://user-images.githubusercontent.com/10403536/80317209-0f3e9980-880b-11ea-9105-654e36c6fb37.png">

People don't see what they type into the name bar which results in names like
<img width="246" alt="Scr 2020-04-26 в 21 53 38" src="https://user-images.githubusercontent.com/10403536/80316804-8a528080-8808-11ea-938e-1a3edb1dfa4f.png">
and
<img width="139" alt="Scr 2020-04-26 в 22 05 14" src="https://user-images.githubusercontent.com/10403536/80317037-0f8a6500-880a-11ea-9d9c-58c4eb9a51a4.png">

**Comment by Ninjaboi8175** [comment]
Note that this has been fixed by someone but not in the stable branch yet.

**Comment by Ninjaboi8175** [comment]
![image](https://user-images.githubusercontent.com/63943484/80371353-64080180-8857-11ea-9cdd-00869a380aac.png)

Just verified that it does seem to have been resolved in the unstable branch. Autumn mentioned it was something to do with forgetting to resize something but looks like when this gets pushed to stable it'll be good.

Edit: Should mention that I only tested it in the editor so if it's after it's built then that's a different matter.
